internal/test/wmcb/wmcb_test.go

```diff
@@ -297,6 +293,25 @@ func (vm *wmcbVM) runTestKubeletUninstall(t *testing.T) {
 	require.NoError(t, err, "TestKubeletUninstall failed")
 }
 
+// isRunning checks if the given service is running
+func (vm *wmcbVM) isRunning(serviceName string) bool {
+	svcCmd := fmt.Sprintf("sc.exe qc %s", serviceName)
+	out, _ := vm.Run(svcCmd, false)
+	return strings.Contains(out, "RUNNING")
+}
+
+// ensureServiceIsRunning creates and starts a given Windows service
+func (vm *wmcbVM) ensureServiceIsRunning(serviceName string, binPath string) error {
```

Comment:
These two helper functions are great and make the code much cleaner, but shouldn't they be in a separate PR? I'm not sure how they relate to this one.

Reply:
My mistake, I see them being used in the next commit